src/components/tabBars.js

```diff
@@ -205,6 +205,7 @@ export function SimpleTabBar({
             'aria-posinset': i + 1, // The first tab is 1
             'aria-setsize': tabs.length,
             'aria-selected': selected,
+            'aria-label': title,
```

Comment:
```suggestion
```

Since `title` is passed as a child to Clickable on line 219, the tab element's content will be the value of `title`. That provides an [accessible name](https://w3c.github.io/accname/#dfn-accessible-name) for the tab element, so `aria-label` isn't necessary here.